src/app/catalog/product/components/product-master/product-master.component.ts

```diff
@@ -24,6 +25,7 @@ export class ProductMasterComponent implements OnInit {
   }
 
   loadProductDetails(id: number) {
-    this.store.dispatch(loadProductDetails({ productId: id }));
+    //product details are now loaded from product-detail component while product-master handles navigation only
```

Comment:
I think we can remove this comment as it refers to previous implementation that readers might not know.